extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -354,6 +354,10 @@ export class AmpAdExit extends AMP.BaseElement {
   assertValidResponseMessage_(responseMessage, eventOrigin) {
     user().assert(responseMessage['message'],
         'Received empty response from 3p analytics frame');
+    user().assert(
```

Comment:
This function is called after the check to `responseMsg['creativeId'] != this.ampAdResourceId_`. The assert here does nothing 😢 
Maybe we can simply ignore the check to 'creativeId', otherwise we will need to move it outside this function

Reply:
Hmm...you're right, the "return" will have executed if creativeId is not defined. I have another idea - testing now, will check in in a few minutes...

Reply:
Ok, how about this?

Reply:
I just realize one thing. If the response does not specify `creativeId`, who should be responsible for throwing this error?
What we have now, the error can be thrown multiple times, because every <amp-ad-exit> who listen to the response will perform the check. I can't think of a really good way to solve this other than create a global listener (which we agree not to)... Do you have any good ideas on this. 

Should we ignore the error, or report it multiple times.

Reply:
Hmm...I agree neither option is ideal, but given the choice between not reporting an error, or reporting it multiple times, I would prefer to report it multiple times.

Reply:
@zhouyx Friendly ping?